src/AudioOutputI2S.cpp

```diff
@@ -104,6 +104,20 @@ bool AudioOutputI2S::SetPinout(int bclk, int wclk, int dout)
 
   return true;
 }
+
+bool AudioOutputI2S::SetPinout(int bclk, int wclk, int dout, int mclk)
```

Comment:
Can you please add `(void) mclk` in the non-ESP32 path?  OTW we get fatal warning/errors.